lib/logger.js

```diff
@@ -10,7 +10,8 @@ const internals = {
 	defaults: {
 		jsonOutput: true,
 		format: 'YYYY-MM-DD HH:mm:ss.SSS',
-		utc: false
+		utc: false,
+		responseFilter: (requestInfo) => requestInfo
```

Comment:
Was a bit confused about the naming especially when the argument is `requestInfo`.
Maybe `requestInfoFilter` would be more appropriate?

Reply:
Agreed. Updated now.